### PR TITLE
Create test case for recipe: internet-of-things

### LINKED ISSUE #1138: Create test case for recipe: internet-of-things

### PR DESCRIPTION
**Pull request by ybyzek**
### Description

#1138

### CODE REVIEW

**Comment by ybyzek** [comment]
Internet of things test case passes

File: _includes/tutorials/inventory/ksql-test/code/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+tutorial-steps/test/manual2.sql
```
Comment (bbejeck) [inline _includes/tutorials/inventory/ksql-test/code/.gitignore:1]: Did you want to include this?

**Review by bbejeck — APPROVED** [review]
One minor comment - otherwise LGTM

File: _includes/tutorials/inventory/ksql-test/code/.gitignore (hunk 1 above)
Comment (ybyzek) [reply _includes/tutorials/inventory/ksql-test/code/.gitignore:1]: Yes, this was missed in another PR.

**Pull request merged**